bug: provider status query requires node flag

```bash
make provider-status
../../akashctl --home "./cache/client" provider status --provider "akash1ypp6mu3y9xeleuwjtpzeaps45jyywfufs0xj7d"
ERROR: no RPC client defined
make: *** [provider-status] Error 1
```

#737 can be a source of this issue

--- a/provider/cmd/root.go
+++ b/provider/cmd/root.go
@@ -2,6 +2,7 @@ package cmd
 
 import (
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/codec"
@@ -12,6 +13,9 @@ func RootCmd(cdc *codec.Codec) *cobra.Command {
 		Use:   "provider",
 		Short: "Akash provider commands",
 	}
+
+	cmd.PersistentFlags().String(flags.FlagNode, "http://localhost:26657", "The node address")
+	_ = viper.BindPFlag(flags.FlagNode, cmd.PersistentFlags().Lookup(flags.FlagNode))
 
 	cmd.AddCommand(sendManifestCmd(cdc))
 	cmd.AddCommand(statusCmd(cdc))

